src/main/web-static/websrc/components/asset-composed-columns-input.html

```
@@ -0,0 +1,86 @@
+<body>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)